### PR TITLE
Fix DeepBox integration tests.

### PR DESCRIPTION
**Pull request by chenkins**
Fixes `testRewriteOverridenFileByRename` and `testSyncConflictDirectoryCreatedLocalAndRemoteIgnoreConflict`.

### CODE REVIEW

File: deepbox/src/main/java/ch/cyberduck/core/deepbox/DeepboxMoveFeature.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public Path move(final Path file, final Path renamed, final TransferStatus statu
             new CoreRestControllerApi(session.getClient()).updateNode(nodeUpdate, sourceId);
             fileid.cache(file, null);
             fileid.cache(renamed, sourceId);
-            return renamed;
+            return renamed.withAttributes(new DeepboxAttributesFinderFeature(session, fileid).find(renamed));
```
Comment (ylangisc) [inline deepbox/src/main/java/ch/cyberduck/core/deepbox/DeepboxMoveFeature.java:71]: Copy original file attributes:
```
return renamed.withAttributes(file.attributes());
```

File: deepbox/src/main/java/ch/cyberduck/core/deepbox/DeepboxDirectoryFeature.java [hunk 2]
```diff
@@ -52,6 +53,9 @@ public DeepboxDirectoryFeature(final DeepboxSession session, final DeepboxIdProv
     @Override
     public Path mkdir(final Path folder, final TransferStatus status) throws BackgroundException {
         try {
+            if(new DeepboxFindFeature(session, fileid).find(folder)) {
```
Comment (ylangisc) [inline deepbox/src/main/java/ch/cyberduck/core/deepbox/DeepboxDirectoryFeature.java:56]: Use `status.isExists()` and trash folder when existing. Refer to `DriveMoveFeature#move`.

Comment (chenkins) [reply deepbox/src/main/java/ch/cyberduck/core/deepbox/DeepboxDirectoryFeature.java:56]: Copied from `DriveDirectoryFeature`.

Comment (dkocher) [reply deepbox/src/main/java/ch/cyberduck/core/deepbox/DeepboxDirectoryFeature.java:56]: `CreateDirectoryWorker` and `TouchWorker` do not determine if a target might already exist. Thus we cannot rely on the `exists` flag in this case but expect implementations to throw failure on conflict. This was introduced in #14168.

**Review by dkocher — CHANGES_REQUESTED** [review]
Add test coverage.

**Comment by dkocher** [comment]
We should also fix `DeepboxCopyFeature` to return attributes of target with 

```
return target.withAttributes(new DeepboxAttributesFinderFeature(session, fileid).toAttributes(copied))
```

**Review by dkocher — APPROVED** [review]

**Pull request merged**